teams/5/settings.html

```diff
@@ -0,0 +1,290 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="settings.css">
+  <title>Settings</title>
+</head>
+
+
+
+<body>
+    <nav class="navbar">
+      <div class="first">
+        <p>nav</p>
+
+        <ul class="top">
+          <p>ul</p>
+          <li><div><i>i</i><a href="#">ul li a</a></div></li>
+          <li><div><i>i</i><a href="#">ul li a</a></div></li>
+          <li><div><i>i</i><a href="#">ul li a</a></div></li>
+          <li><div><i>i</i><a href="#">ul li a</a></div></li>
+          <li><div><i>i</i><a href="#">ul li a</a></div></li>
+          <li><div><i>i</i><a href="#">ul li a</a></div></li>
+        </ul> 
+
+        <ul class="bottom">
+          <p>ul</p>
+          <li><i>i</i><a href="#">ul li a</a></li>
+          <li><i>i</i><a href="#">ul li a</a></li>
+          <li>    
+            <img src="ms.jpg" alt="img" width="50px" height="50px">
+          </li>
+        </ul> 
+      </div>
+      <div class="second">
+        <ul>
+          <p>ul</p>
+          <li><div class="row"><i>i</i><a href="#">ul li a</a> <span>span</span> </div></li>
+          <li><div class="row"><i>i</i><a href="#">ul li a</a> <span>span</span>  </div></li>
+          <li><div class="row"><i>i</i><a href="#">ul li a</a> <span>span</span>  </div></li>
+          <li><div class="row"><i>i</i><a href="#">ul li a</a> <span>span</span>  </div></li>
+          <li><div class="row"><i>i</i><a href="#">ul li a</a> <span>span</span>  </div></li>
+          <li><div class="row"><i>i</i><a href="#">ul li a</a> <span>span</span>  </div></li>
+          <li><div class="row"><i>i</i><a href="#">ul li a</a> <span>span</span>  </div></li>
+   
+        </ul>  
+       
+         <ul>       
+          <p>ul</p>  
+          <li><div class="spaceTopTwo"><i>i</i><a href="#">ul li a</a></div></li>
+          <li><div><i>i</i><a href="#">ul li a</a></div></li>
+         <li>
+           <div class="row"></div>
+             <div  class="spaceTopTwo">
+             <h2>h2</h2>
+             <p>p</p>
+             </div>
+           <i>i</i>
+          </div>
+        </li> 
+        </ul>  
+      </div>
+    </nav> 
+
+ <main>
+    <div class="mainHead">
+     <p>main div</p>
+    </div>
+
+  <header>
```

Comment:
Should we wrap this whole part to <section>?

Reply:
dont understand your question please clarify, wrap in what?